test/artifact_test.js

```diff
@@ -15,6 +15,15 @@ suite('Artifacts', function() {
   var {Netmask}     = require('netmask');
   var assume        = require('assume');
   var helper        = require('./helper');
+  var fs            = require('fs');
+  var crypto        = require('crypto');
+  var randbytes     = require('randbytes');
+  var tmp           = require('tmp');
+  var remoteS3      = require('remotely-signed-s3');
+  var qs            = require('querystring');
+  var urllib        = require('url');
```

Comment:
just saying other places we've often used `URL = require('url')` to avoid the problem with `url` variable...

Note, in golang I have the same issue, and have started prefering `u` instead of `url` as variable name.

Regardless, this is nits...